imports/core/api/models/proposals_pubs.js

```diff
@@ -1,5 +1,10 @@
-import { Proposals } from './proposals';
-import { Counts } from 'meteor/tmeasday:publish-counts';
+import { Meteor } from 'meteor/meteor'
+import { Proposals } from './proposals'
+import { Counts } from 'meteor/tmeasday:publish-counts'
```

Comment:
Can we make some conventions about imports?
I think semicolons at end of line is necessary and better to split imports from local files and packages by one empty line.